samples/bluetooth/peripheral/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ target_sources(app PRIVATE
   src/main.c
   ../gatt/hrs.c
   ../gatt/dis.c
-  ../gatt/bas.c
+  $ENV{ZEPHYR_BASE}/subsys/bluetooth/services/bas.c
```

Comment:
This shouldn't be necessary, instead the application should just select the Kconfig option to enable BAS so in this case it would be part of prj.conf.

Reply:
I created a commit where I first moved the service, then another where I created the Kconfig entry and updated the service to use the sensor interface.

Reply:
@lemrey can you please squash the 2 commits so we don't see this?